bin/mocha

```diff
@@ -123,7 +121,8 @@ debug('final node args', nodeArgs);
 const args = [].concat(
   unparseNodeFlags(nodeArgs),
   mochaPath,
-  unparse(mochaArgs, {alias: aliases})
+  '--preParsedJsonOptions',
+  JSON.stringify(mochaArgs)
 );
```

Comment:
This way duplicate alias options don't get eliminated. With `unparse()` we get a clean array without any aliases, so I'm not sure wether we can do without  `unparse()` as easily.

It's a very good idea not to parse a second time in "/cli.cli.js".

Reply:
I don't think it's necessary to remove duplicate alias options using this method.  They won't pass through yargs' parser again; the object is passed straight to mocha.

When mocha v6 re-parses the output of yargs-unparser, it creates those duplicate aliases all over again.  So either way, the output object contains aliases, and mocha is built to handle it.

Reply:
Yes, I think the `mocha` instance looks quite messy with all those aliases, but it seems to be able to handle them correctly.

> [...]; the object is passed straight to mocha.

This could be to early, since the coerce function of `reporter-option` hasn't run yet?